frontend/src/Editor/Components/components.js

```diff
@@ -618,6 +618,7 @@ export const componentTypes = [
     },
     properties: {
       label: { type: 'code', displayName: 'Label' },
+      defaultValue: { type: 'toggle', displayName: 'Default value' },
```

Comment:
Lets call it default status instead of default value

```suggestion
      defaultValue: { type: 'toggle', displayName: 'Default status' },
```